ee/server/service/software_installers.go

```diff
@@ -108,3 +109,83 @@ func (svc *Service) DeleteSoftwareInstaller(ctx context.Context, id uint) error
 
 	return nil
 }
+
+func (svc *Service) GetSoftwareInstallerMetadata(ctx context.Context, installerID uint) (*fleet.SoftwareInstaller, error) {
+	if err := svc.authz.Authorize(ctx, &fleet.SoftwareInstaller{}, fleet.ActionRead); err != nil {
```

Comment:
will this check always fail for people on a team due to the lack of `TeamID`?

https://github.com/fleetdm/fleet/blob/298e7eecc60939934e3d6084c44cc9287525a4dc/server/authz/policy.rego#L660-L667

Reply:
You're right! Let me know what you think of the updated approach.